env.cpu.yml

```diff
@@ -3,7 +3,7 @@ dependencies:
     - -f https://download.pytorch.org/whl/torch_stable.html
     - -f https://pytorch-geometric.com/whl/torch-1.8.0+cpu.html
     - dgl==0.6.*
-    - torch==1.8.1+cpu
-    - torchvision==0.9.1+cpu
+    - torch==1.8.0+cpu
+    - torchvision==0.9.0+cpu
     - torch-scatter
```

Comment:
```
    - torch-scatter==2.0.6
    - torch-sparse==0.6.9
```

Could try this

Reply:
@rusty1s suggested in https://github.com/deepchem/deepchem/pull/2541 that we should use 1.8.0 so I think that's the latest? But not sure

Reply:
Trying pinning `torch-scatter` and `torch-sparse` as you suggest now

Reply:
PyTorch 1.8.1 is supported, so feel free to install it. However, you need to use the PyG 1.8.0 wheel page for that as well:
```
-f https://pytorch-geometric.com/torch-1.8.0+cpu.html
```